src/gui/windows/delete_world_confirmation.zig

```diff
@@ -18,6 +18,10 @@ const padding: f32 = 8;
 
 var deleteWorldName: []const u8 = "";
 
+pub fn init() void {
+	deleteWorldName = main.globalAllocator.dupe(u8, "");
```

Comment:
That's equivalent to `deleteWorldName = "";`, since zero-length allocations (and frees) are special-cased to do nothing.

Reply:
Didn't know that, then let it be as suggested

```suggestion
	deleteWorldName = "";
```